src/kernel/kmain.zig

```diff
@@ -28,7 +29,14 @@ var kernel_vmm: vmm.VirtualMemoryManager(arch.VmmPayload) = undefined;
 
 // This is for unit testing as we need to export KERNEL_ADDR_OFFSET as it is no longer available
 // from the linker script
+// These will need to be kept up to date with the debug logs in the mem init.
 export var KERNEL_ADDR_OFFSET: u32 = if (builtin.is_test) 0xC0000000 else undefined;
+export var KERNEL_STACK_START: u32 = if (builtin.is_test) 0xc013c000 else undefined;
+export var KERNEL_STACK_END: u32 = if (builtin.is_test) 0xc0140000 else undefined;
+export var KERNEL_VADDR_START: u32 = if (builtin.is_test) 0xC0100000 else undefined;
+export var KERNEL_VADDR_END: u32 = if (builtin.is_test) 0xc0140000 else undefined;
+export var KERNEL_PHYSADDR_START: u32 = if (builtin.is_test) 0x100000 else undefined;
+export var KERNEL_PHYSADDR_END: u32 = if (builtin.is_test) 0x140000 else undefined;
```

Comment:
How come we need these here now?

Reply:
If they are needed for unit testing, then don't need to add them. Plus some of them are needed

Reply:
Ok, we could keep them here for now, but I think we should clean this sort of thing up in another PR since they're used all over the place. Perhaps we could move them to mem.zig and abstract them away with functions.